python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -75,7 +75,7 @@ def prepare(self, project: "DbtProject") -> None:
 @experimental
 class DbtProject(DagsterModel):
     project_dir: Path
-    target_dir: Path
+    target_path: Path
     manifest_path: Path
     state_dir: Optional[Path]
```

Comment:
should it be `state_path` since the behavior is project relative like `target_path`?

Reply:
Good call, also added it in this PR.